aws/local.yml

```diff
@@ -26,6 +26,7 @@
     - verifier
```

Comment:
The oauth role is still here to do some config, which really should be moved fully into the auth role, but not this round.